en/mkdocs.yml

```diff
@@ -335,6 +335,7 @@ nav:
                       - Multi Factor Authentication for Publisher and Developer Portals: develop/extending-api-manager/saml2-sso/multi-factor-authentication-mfa-for-publisher-and-developer-portals.md
           - Customizations:
                 - Customizing the Developer Portal:
+                      - Customize Developer Portal Credential Preferences: develop/customizations/customizing-the-developer-portal/customize-developer-portal-credentials.md
```

Comment:
if we change as per above, we'll have to refelect the change here too.

Reply:
Changed